### DISCUSSION TITLE
5th rust-libp2p community call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#2998] [General]
Hi everyone,

The rust-libp2p maintainers (@thomaseizinger @elenaf9 and I) will meet **next week Tuesday**. See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-community-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Items on the agenda thus far:

- https://github.com/libp2p/rust-libp2p/discussions/2938
- https://github.com/libp2p/rust-libp2p/pull/2997/

See you all next week.

**Comment by umgefahren** [comment]
Sadly I couldn’t attend. Are the results documented somewhere?

secondly: Is the community call every two weeks at 9:00-9:45 am UTC?

**Reply by thomaseizinger** [reply]
Yes, typically it is every two weeks!

@mxinden I am afraid I can't do Mondays anymore starting from the 24th. Can we move it permanently to Tuesdays?

**Reply by mxinden** [reply]
Moved to Tuesday @thomaseizinger.

**Comment by mxinden** [comment]
@umgefahren a couple of notes from the meeting:

- João starting yesterday (35-40h)
    - https://github.com/jxs
- Bevy-style NetworkBehaviours / plugins
    - https://github.com/libp2p/rust-libp2p/discussions/2938
- Roadmap
    - https://github.com/libp2p/rust-libp2p/pull/2997/
    - https://github.com/libp2p/go-libp2p/pull/1784
- Testground multi-dimensions
    - https://github.com/libp2p/test-plans/issues/53
